Scripts/Plot_CarAndTrailer.m

```diff
@@ -1,56 +1,140 @@
+% Plota_carAndTrailer.m
+%   This script plot steering angle, jackknife angle
+%   and car and trailer displacement.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%
 close all;
-x1=ScopeData.signals.values(1:151,1),'DisplayName','ScopeData.signals.values(1:151,1)','YDataSource','ScopeData.signals.values(1:151,1)'
-y1=ScopeData.signals.values(1:151,2),'DisplayName','ScopeData.signals.values(1:151,2)','YDataSource','ScopeData.signals.values(1:151,2)'
-xh=ScopeData.signals.values(1:151,3),'DisplayName','ScopeData.signals.values(1:151,3)','YDataSource','ScopeData.signals.values(1:151,3)'
-yh=ScopeData.signals.values(1:151,4),'DisplayName','ScopeData.signals.values(1:151,4)','YDataSource','ScopeData.signals.values(1:151,4)'
-x2=ScopeData.signals.values(1:151,5),'DisplayName','ScopeData.signals.values(1:151,5)','YDataSource','ScopeData.signals.values(1:151,5)'
-y2=ScopeData.signals.values(1:151,6),'DisplayName','ScopeData.signals.values(1:151,6)','YDataSource','ScopeData.signals.values(1:151,6)'
-phi=ScopeData.signals.values(1:151,7),'DisplayName','ScopeData.signals.values(1:151,7)','YDataSource','ScopeData.signals.values(1:151,7)'
-gamma=ScopeData.signals.values(1:151,8),'DisplayName','ScopeData.signals.values(1:151,8)','YDataSource','ScopeData.signals.values(1:151,8)'
-jkk=ScopeData_jkk.signals.values(1:151,1),'DisplayName','ScopeData.signals.values(1:151,2)','YDataSource','ScopeData.signals.values(1:151,1)'
+x1=ScopeData.signals.values(1:151,1);'DisplayName';'ScopeData.signals.values(1:151,1)';'YDataSource';'ScopeData.signals.values(1:151,1)';
+y1=ScopeData.signals.values(1:151,2);'DisplayName';'ScopeData.signals.values(1:151,2)';'YDataSource';'ScopeData.signals.values(1:151,2)';
+xh=ScopeData.signals.values(1:151,3);'DisplayName';'ScopeData.signals.values(1:151,3)';'YDataSource';'ScopeData.signals.values(1:151,3)';
+yh=ScopeData.signals.values(1:151,4);'DisplayName';'ScopeData.signals.values(1:151,4)';'YDataSource';'ScopeData.signals.values(1:151,4)';
+x2=ScopeData.signals.values(1:151,5);'DisplayName';'ScopeData.signals.values(1:151,5)';'YDataSource';'ScopeData.signals.values(1:151,5)';
+y2=ScopeData.signals.values(1:151,6);'DisplayName';'ScopeData.signals.values(1:151,6)';'YDataSource';'ScopeData.signals.values(1:151,6)';
+phi=ScopeData.signals.values(1:151,7);'DisplayName';'ScopeData.signals.values(1:151,7)';'YDataSource';'ScopeData.signals.values(1:151,7)';
+gamma=ScopeData.signals.values(1:151,8);'DisplayName';'ScopeData.signals.values(1:151,8);''YDataSource';'ScopeData.signals.values(1:151,8)';
+jkk=ScopeData_jkk.signals.values(1:151,1);'DisplayName';'ScopeData.signals.values(1:151,1);''YDataSource';'ScopeData.signals.values(1:151,1)';
+steering=ScopeData_SteeringAngle.signals.values(1:151,1);'DisplayName';'ScopeData.signals.values(1:151,1)';'YDataSource';'ScopeData.signals.values(1:151,1)';
 
 figure('Name','Car and Trailer model','NumberTitle','off');
+subplot(3,2,1);
+p1=plot(steering,'r','linewidth',2);
+grid on;
```

Comment:
At my experience grid on or hold on not generate any output to work space or log, but anyway, doesn't affect the behavior, just for your information